### PR TITLE
update metadata auth docs with an example of the auth header

### PR DESCRIPTION
**Pull request by Jstein77**
## What are you changing in this pull request and why?
**What changed:** Added an example of the header key with a token prefix for authenticating with the Metadata API.

**Why:** I tried to authenticate to the Metadata API but kept getting an error. I needed to include the token prefix in the Auth header. This was mentioned earlier in the doc, but there was not a clear example. 
## Checklist

-  [x] Review the [Content style guide](https://github.com/dbt-labs/docs.getdbt.com/blob/current/contributing/content-style-guide.md) and [About versioning](https://github.com/dbt-labs/docs.getdbt.com/blob/current/contributing/single-sourcing-content.md#adding-a-new-version) so my content adheres to these guidelines.
- [x] Add a checklist item for anything that needs to happen before this PR is merged, such as "needs technical review" or "change base branch."

### CODE REVIEW

File: website/docs/docs/dbt-cloud-apis/metadata-querying.md [hunk 1]
```diff
@@ -44,7 +44,7 @@ Refer to the [Apollo explorer documentation](https://www.apollographql.com/docs/
 1. Access the [GraphQL API explorer](https://metadata.cloud.getdbt.com/graphql) and select fields you'd like query. 
 2. Go to **Variables** at the bottom of the explorer and replace any `null` fields with your unique fields.
 3. [Authenticate](https://www.apollographql.com/docs/graphos/explorer/connecting-authenticating#authentication) via bearer auth with `YOUR_TOKEN`. Go to **Headers** at the bottom of the explorer and select **+New header**.
```
Comment (mirnawong1) [inline website/docs/docs/dbt-cloud-apis/metadata-querying.md]:
```suggestion
3. [Authenticate](https://www.apollographql.com/docs/graphos/explorer/connecting-authenticating#authentication) via bearer auth with `YOUR_AUTH_TOKEN`. Go to **Headers** at the bottom of the explorer and select **+New header**.
```

File: website/docs/docs/dbt-cloud-apis/metadata-querying.md [hunk 2]
```diff
@@ -42,10 +42,16 @@ You can run ad-hoc queries directly in the [GraphQL API explorer](https://metada
...
 
 1. Access the [GraphQL API explorer](https://metadata.cloud.getdbt.com/graphql) and select fields you'd like query. 
+
 2. Go to **Variables** at the bottom of the explorer and replace any `null` fields with your unique fields.
-3. [Authenticate](https://www.apollographql.com/docs/graphos/explorer/connecting-authenticating#authentication) via bearer auth with `YOUR_TOKEN`. Go to **Headers** at the bottom of the explorer and select **+New header**.
-4. Select **Authorization** in the **header key** drop-down list and enter your Bearer auth token in the **value** field.
+
+3. [Authenticate](https://www.apollographql.com/docs/graphos/explorer/connecting-authenticating#authentication) via Bearer auth with `YOUR_TOKEN`. Go to **Headers** at the bottom of the explorer and select **+New header**.
+
+4. Select **Authorization** in the **header key** drop-down list and enter your Bearer auth token in the **value** field. Remember to include the Token prefix. Your header key should look like this `{"Authorization": "Bearer <YOUR_TOKEN>}`.
```
Comment (mirnawong1) [inline website/docs/docs/dbt-cloud-apis/metadata-querying.md:50]:
```suggestion
4. Select **Authorization** in the **header key** drop-down list and enter your Bearer auth token in the **value** field. Remember to include the Token prefix. Your header key should look like this `{"Authorization": "Bearer <auth_token>}`.
```

File: website/docs/docs/dbt-cloud-apis/metadata-querying.md [hunk 3]
```diff
@@ -42,10 +42,16 @@ You can run ad-hoc queries directly in the [GraphQL API explorer](https://metada
...
 
 1. Access the [GraphQL API explorer](https://metadata.cloud.getdbt.com/graphql) and select fields you'd like query. 
+
 2. Go to **Variables** at the bottom of the explorer and replace any `null` fields with your unique fields.
-3. [Authenticate](https://www.apollographql.com/docs/graphos/explorer/connecting-authenticating#authentication) via bearer auth with `YOUR_TOKEN`. Go to **Headers** at the bottom of the explorer and select **+New header**.
-4. Select **Authorization** in the **header key** drop-down list and enter your Bearer auth token in the **value** field.
+
+3. [Authenticate](https://www.apollographql.com/docs/graphos/explorer/connecting-authenticating#authentication) via Bearer auth with `YOUR_TOKEN`. Go to **Headers** at the bottom of the explorer and select **+New header**.
+
+4. Select **Authorization** in the **header key** drop-down list and enter your Bearer auth token in the **value** field. Remember to include the Token prefix. Your header key should look like this `{"Authorization": "Bearer <auth_token>}`.
```
Comment (mirnawong1) [inline website/docs/docs/dbt-cloud-apis/metadata-querying.md]: love this thank you! changed it to 'your_token' for consistency across the page.
```suggestion
4. Select **Authorization** in the **header key** drop-down list and enter your Bearer auth token in the **value** field. Remember to include the Token prefix. Your header key should look like this `{"Authorization": "Bearer <YOUR_TOKEN>}`.
```

**Review by mirnawong1 — DISMISSED** [review]
this is awesome, thank you for opening up this pr to add this clarifying detail @Jstein77 ! I've also added an image to model how to add it, which i originally intended to add but went back on that. 

only one small non blocking suggestion, let me know what you think!

**Comment by Jstein77** [comment]
Beautiful! These are great suggestions.

**Review by mirnawong1 — APPROVED** [review]

**Comment by mirnawong1** [comment]
So glad we got your first docs pr merged @Jstein77 - thank you for opening this up and improving the docs!

**Pull request merged**